src/og/control/LayerSwitcher.js

```diff
@@ -5,20 +5,24 @@
 "use strict";
 
 import { Control } from "./Control.js";
+import { elementFactory, allMenuBtnOFF, allDialogsHide, btnClickHandler } from "./UIhelpers.js";
 
 /**
- * Simple(OpenLayers like)layer switcher, includes base layers, overlays, geo images etc. groups.
+ * Advanced :) layer switcher, includes base layers, overlays, geo images etc. groups.
+ * Double click for zoom, drag-and-drop to change zIndex
  * @class
  * @extends {Control}
  * @param {Object} [options] - Control options.
  */
 class LayerSwitcher extends Control {
     constructor(options) {
         super(options);
-
         this.dialog = null;
-        this.baseLayersDiv = null;
-        this.overlaysDiv = null;
+        this.baseLayersContainer = null;

```

Comment:
Maybe for future things, I suggest using prefix $ for dom variables, like this.$baseLayerContainer or even $layers. It's not needed to change now, but if you find it good, we can use this rule.

Reply:
Maybe DOM_ would be better?